test/functional/isVisible.js

```diff
@@ -0,0 +1,58 @@
+const { assert } = require('chai');
+const firefox = require('selenium-webdriver/firefox');
+const webdriver = require('selenium-webdriver');
+const { ancestors, isDomElement, isVisible, toDomElement } = require('../../utilsForFrontend');
```

Comment:
The rest of the codebase has no spaces inside the braces. That would be a good linter rule.

Reply:
I added the eslint rules for this; also I noticed my test files weren't being linted, so I amended `make lint`. Unfortunately, I wasn't able to get the `.mjs` and my `test/functional` `.js` files linted in a single eslint command, so I added a second command.